pages/index.html

```diff
@@ -1,12 +1,196 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/home.css">
+    <link rel="stylesheet" href="../styles/about.css">
+    <link rel="stylesheet" href="../styles/portfolio.css">
+    <link rel="stylesheet" href="../styles/contact.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Grape+Nuts&family=Source+Code+Pro:wght@300&family=Ubuntu:wght@300;400&display=swap"
+        rel="stylesheet">
+    <script src="https://kit.fontawesome.com/813a89c764.js" crossorigin="anonymous"></script>
+    <title>Lin's Website</title>
 </head>
-<body>
-  
+
+<body id="main">
+    <header>
+        <nav class="navbar">
+            <a href="#home" class="nav-name">Lin Liu</a>
+            <ul class="nav-menu">
+                <li class="nav-link"><a href="#home"> Home </a></li>
+                <li class="nav-link"><a href="#about"> About </a></li>
+                <li class="nav-link"><a href="#portfolio"> Portfolio </a></li>
+                <li class="nav-link"><a href="#" class="media-content" onclick="openNav()"> Contact </a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="container">
+        <section class="home" id="home">
+            <aside class="side-bar left-side-bar"></aside>
+            <div class="content">
+                <div class="content-hello">
+                    <h1 id="hello">Hello, World!</h1>
+                </div>
+                <article>
+                    <h2 class="content-item" id="name">I am Lin</h2>
+                    <h2 class="content-item" id="desc">{"A Software Developer in training"}</h2>
+                </article>
+            </div>
+            <aside class="side-bar right-side-bar">
+            </aside>
+            <footer>
+                <img src="../assets/Images/next-page.png" alt="next-page">
+            </footer>
+        </section>
+        <section class="about" id="about">
+            <aside class="side-bar left-side-bar"></aside>
+            <div class="content">
+                <div class="content-item skills">
+                    <h3>SKILLS</h3>
+                    <ul class="my-skills">
+                        <li class="skill ">
+                            <i class="fa-brands fa-python fa-3x"></i>
```

Comment:
I'm not a fan of font-awesome because it introduces these tags with no meaning in your code.

It's commonly used though so seems to just be a me thing.  :-P